x/multi-staking/keeper/store.go

```diff
@@ -73,27 +73,6 @@ func (k Keeper) ValidatorMultiStakingCoinIterator(ctx sdk.Context, cb func(valAd
 	}
 }
 
-func (k Keeper) GetIntermediaryDelegatorKey(ctx sdk.Context, multiStakerAddr sdk.AccAddress) sdk.AccAddress {
```

Comment:
we should replace this with a store for multistaker accounts instead of removing it

Reply:
because delegators don't stake by intermediary account any more and we can query delegations by delegator and validator address so this store shoule be removed